website/src/css/_benefits-section.scss

```diff
@@ -0,0 +1,24 @@
+.benefits-header {
+  font-family: 'Nunito Sans', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 58px;
+  line-height: 70px;
+  color: #131419;
+  margin-top: 125px;
+  margin-bottom: 125px;
+}
+
+[data-theme='dark'] .benefits-header {
+  color: #ffffff;
+}
+
+.social-buttons {
+  justify-content: flex-end;
+  display: flex;
+}
+
+.flex-jc-fe {
```

Comment:
Does it really belong to the benefits section? Looks generic on a first glance.

Reply:
@noomorph  - This is being using just in the Homepage SubHeader for the social icons. It's why I put it here. Is it not correct?
![image](https://user-images.githubusercontent.com/34761997/216348853-4e32c25e-1563-4b02-acfc-f470c67af6fc.png)